ex2/verify-result.sh

```diff
@@ -52,6 +52,13 @@ if [[ $string != *"ESCAPED_REFERENCE=https://172.17.0.1:80"* ]]; then
 #echo found: $string
 fi
 
+# test if configmap has been created
+string=$(kubectl -n$NS get cm)
+if [[ $string != *cm1* ]]; then
```

Comment:
pls. so that I learn perhaps more bash:
why the asterisk *cm1*
what's the difference to "cm1"

Looks like some regex.
If 'cm1' is requested then it should check for exactly cm1

Reply:
We should change *cm1* to *"cm1"*. This way it will check for the presence of the string cm1.